web3/web3/method.py

```diff
@@ -0,0 +1,127 @@
+import utils.utils
+import errors
+
+
+class Method:
```

Comment:
Fancy, I wasn't aware of that.

I would however like to support python 2.7 so can we add it?